sig-network/charter.md

```diff
@@ -26,9 +26,9 @@ The following topics fall under ownership of this SIG:
 #### Code, Binaries and Services
 
 - Services
-  - APIs for defining and grouping networking endpoints.
-  - APIs for defining L3/4 loadbalancing.
-  - Reference implementation (kube-proxy).
+  - APIs for defining and grouping networking (e.g. [EndpointSlices], or the older [Endpoints] API)
```

Comment:
You should keep the word "endpoints" there. The APIs are not for grouping networking, they're for grouping networking endpoints.

Also, all of these are probably better as "i.e." than "e.g."